numpy/ma/core.py

```diff
@@ -93,6 +93,33 @@
 class MaskedArrayFutureWarning(FutureWarning):
     pass
 
+def _deprecate_argsort_axis(arr, axis):
+    """
+    Takes the array argsort was called upon, and the axis argument
```

Comment:
Should follow usual docstring standard. This is really something for the `Parameters` section.